src/Registration/Common/RegisterImageToAnotherUsingLandmarks/moving.png.sha512

```diff
@@ -1 +1 @@
-9411c2f629a4c1d833b517c85527e85990d85d4e5f45edc52f5b5ab6b14ff1ffa54ad7b28ce56023456a4602c25e49b2acb81d216b4620efde3b615655f4b474
+376682f715f8d870b1bf91ae037914447220b5d8a11546daaf0df0fe1eec7bc35f5a53f563287849740abb7fe74b4b4c002b1c4d1be646acc614bb4b018a9e82
```

Comment:
Why do the images need to be updated?

Reply:
These snuck in there somehow. Now removed.